docs/source/configuration/settings-reference.md

```diff
@@ -232,6 +232,8 @@ styleClassNameConverters
     data to actual class names. You can customize the generated classname by
     registering fieldnames with names such as `<fieldname>:<converterName>`,
     where the converter is registered here.
+```
```

Comment:
The glossary has nested directives. I wonder that the build does not complain and asks for increased number of ticks.

Reply:
The nested directives are indented, thus their context starts new, and are not really "nested". If they were at the same level of indentation, then the outer directives would need to have extra backticks for their nested directives.

BTW, I had the same question, and my testing confirmed this.